src/transformers/generation/tf_utils.py

```diff
@@ -849,6 +984,7 @@ def generate(
                 length_penalty=generation_config.length_penalty,
                 early_stopping=generation_config.early_stopping,
                 logits_processor=logits_processor,
+                output_scores=generation_config.output_scores,
```

Comment:
(this one was missing, which means that users of previous versions couldn't return the scores with beam search 🤦 )